Gemfile

```diff
@@ -117,6 +117,7 @@ gem 'dry-configurable', '0.13.0'
 gem 'dry-container', '0.9.0'
 gem 'devise-security'
 gem 'file_validators'
+gem 'actiontext', '6.1.7.9'
```

Comment:
@jacobkagon Why are we not upgrading the rails gem itself and why specifically actiontext?

Reply:
@saikumar9 I wasn't sure if we wanted to upgrade Rails.

Reply:
Let's upgrade Rails and run the `bundler-audit` check to see if there are any failures.